babel.config.js

```diff
@@ -56,7 +56,7 @@ const config = {
 	] ),
 	overrides: [
 		{
-			test: './client/gutenberg/extensions',
+			test: [ './client/gutenberg/extensions', './packages/jetpack-blocks' ],
```

Comment:
I think this will work out because we're using Calypso's build right now (breaking out of the package boundary with `../../…`.

We'll need the build to get this right when it is published - #31281